src/hyp3_autorift/process.py

```diff
@@ -319,6 +330,40 @@ def apply_landsat_filtering(reference_path: str, secondary_path: str) \
     return reference_path, reference_zero_path, secondary_path, secondary_zero_path
 
 
+def get_lon_lat_from_ncfile(ncfile):
+    with Dataset(ncfile) as ds:
+        var = ds.variables['img_pair_info']
+        return var.latitude, var.longitude
+
+
+def point_to_prefix(dir_path: str, lat: float, lon: float) -> str:
+    """
+    Returns a string (for example, N78W124) for directory name based on
+    granule centerpoint lat,lon
+    """
+    NShemi_str = 'N' if lat >= 0.0 else 'S'
+    EWhemi_str = 'E' if lon >= 0.0 else 'W'
+
+    outlat = int(10*np.trunc(np.abs(lat/10.0)))
+    if outlat == 90:  # if you are exactly at a pole, put in lat = 80 bin
+        outlat = 80
+
+    outlon = int(10*np.trunc(np.abs(lon/10.0)))
+
+    if outlon >= 180:  # if you are at the dateline, back off to the 170 bin
+        outlon = 170
+
+    dirstring = os.path.join(dir_path, f'{NShemi_str}{outlat:02d}{EWhemi_str}{outlon:03d}')
+    return dirstring
+
+
+def get_opendata_prefix(file: Path, scene: str):
```

Comment:
you can also get the scene name from the file name
```python
scene = file.name.split('_X_')[0]
```
which is better since we sort the granules on input. Either that, or when this is called, you should pass it `g1`:
https://github.com/ASFHyP3/hyp3-autorift/blob/develop/src/hyp3_autorift/process.py#L538

Reply:
done!